schema_utils.py

```diff
@@ -221,6 +240,17 @@ class _Validators(object):
     """
     @classmethod
     def get(cls, validator_id):
+        """Gets the value of the validator_id of the object.
+
+        Args:
+            validator_id: str. That contains the attribute's name.
```

Comment:
This is vague. What does "the attribute" refer to?

I think you can just say "The validator ID." here. It's clear given the amended docstring I suggested above.

Reply:
Done